src/transformers/models/imagegpt/modeling_imagegpt.py

```diff
@@ -992,11 +992,12 @@ def forward(
         ... )
 
         >>> clusters = feature_extractor.clusters
-        >>> n_px = feature_extractor.size
+        >>> n_px_h = feature_extractor.size["height"]
+        >>> n_px_w = feature_extractor.size["width"]
 
```

Comment:
Could we use height and width here? Equally long and much more understandable.

Reply:
Yep 👍 Changed now.